src/twisted/internet/interfaces.py

```diff
@@ -1457,7 +1465,7 @@ def fireSystemEvent(eventType):
         'persist'.
         """
 
-    def addSystemEventTrigger(phase, eventType, callable, *args, **kw):
+    def addSystemEventTrigger(phase, eventType, callableObj, *args, **kw):
```

Comment:
Separate PR?

```
$ grep -r "def addSystemEventTrigger(" src/
src/twisted/test/test_adbapi.py:    def addSystemEventTrigger(self, phase, event, trigger):
src/twisted/test/test_twistd.py:    def addSystemEventTrigger(self, *args, **kw):
src/twisted/test/test_task.py:    def addSystemEventTrigger(self, phase, event, callable, *args):
src/twisted/internet/interfaces.py:    def addSystemEventTrigger(phase, eventType, callable, *args, **kw):
src/twisted/internet/base.py:    def addSystemEventTrigger(self, _phase, _eventType, _f, *args, **kw):
src/twisted/internet/testing.py:    def addSystemEventTrigger(self, phase, eventType, callable, *args, **kw):
```

Reply:
I've moved this to: https://github.com/twisted/twisted/pull/1325